### PR TITLE
[fix-#3680][service]  Zookeeper does not start, printed logs have no error messages

### LINKED ISSUE #3680: [Bug][API] Zookeeper does not start, printed logs have no error messages, Api services can start but cannot be accessed.
Zookeeper does not start, printed logs have no error messages, Api services can start but cannot be accessed.
Zookeeper没有启动，打印的日志没有错误信息，Api服务能启动但是不能访问。

### PR DESCRIPTION
**Pull request by dailidong**
fix https://github.com/apache/incubator-dolphinscheduler/issues/3680, if can't connect zookeeper successfully, will throw exception, instead of waiting all the time 

and delete no useful TheadUtilsTest

this closes #3680

### CODE REVIEW

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/threadutils/ThreadUtilsTest.java [hunk 1]
```diff
@@ -37,12 +37,12 @@
     @Test
     public void testNewDaemonFixedThreadExecutor() {
         // create core size and max size are all 3
-        ExecutorService testExec = ThreadUtils.newDaemonFixedThreadExecutor("test-exec-thread",10);
+        ExecutorService testExec = ThreadUtils.newDaemonFixedThreadExecutor("test-exec-thread",3);
```
Comment (yangyichao-mango) [inline dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/threadutils/ThreadUtilsTest.java:40]:
```suggestion
        ExecutorService testExec = ThreadUtils.newDaemonFixedThreadExecutor("test-exec-thread", 3);
```

File: dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/threadutils/ThreadUtilsTest.java [hunk 2]
```diff
@@ -77,13 +77,12 @@ public void run() {
...
                     globalTimer.add(Calendar.SECOND, 1);
                 }
-                System.out.println("time is " + System.currentTimeMillis());
             }
         };
         scheduleService.scheduleAtFixedRate(schedulerTask, 2, 10, TimeUnit.SECONDS);
         assertFalse(scheduleService.isShutdown());
         try {
-            Thread.sleep(60000);
+            Thread.sleep(2000);
```
Comment (CalvinKirs) [inline dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/threadutils/ThreadUtilsTest.java:85]: Sleep is not recommended in the test code, maybe we should try to use Awaitility to achieve the same effect?

**Comment by dailidong** [comment]
@CalvinKirs  @yangyichao-mango ， I reviewed this UT test，no useful, just JDK ThreadPool test, so I decide to delete it

**Review by CalvinKirs — APPROVED** [review]
+1

**Pull request merged**